x/shield/keeper/purchase.go

```diff
@@ -217,7 +217,7 @@ func (k Keeper) RemoveExpiredPurchasesAndDistributeFees(ctx sdk.Context) {
 					purchaseList.Entries[i].ServiceFees = types.InitMixedDecCoins()
 
 					originalStaking := k.GetOriginalStaking(ctx, entry.PurchaseID)
-					if !originalStaking.IsZero() && ctx.BlockHeight() > common.Update1Height {
+					if !originalStaking.IsZero() && ctx.BlockHeight() >= common.Update1Height {
```

Comment:
I think before height there is no staking shield regardless?

Reply:
Right. Fixed.